src/WidgetRegistry.ts

```diff
@@ -53,6 +53,13 @@ export function isWidgetBaseConstructor(item: any): item is WidgetBaseConstructo
 	return Boolean(item && item._type === WIDGET_BASE_TYPE);
 }
 
+function getEventType(label: RegistryLabel): RegistryLabel {
```

Comment:
Well and that is the more macro question, if the event type for a loaded event emitted from the widget registry for a string label of `foo` is `loaded:foo` but for `Symbol('foo')` is `@@foo` then we can only ever have one type of event for symbol labels from `WidgetRegistry` forever.  That was part of my argument that actually the string label being part of the event type felt more like other meta data on the event than an actual event type.  Like `{ type: 'loaded', label }`, but I wasn't sure of the downstream impacts of that.